src/devices/Blinkt/samples/README.md

```diff
@@ -0,0 +1,3 @@
+﻿# TODO: This needs to be determined
```

Comment:
You can delete this file, the one in the parent directory (which includes usage information) is enough.